smallslive/templates/artist_dashboard/artist-dashboard-events.html

```diff
@@ -60,6 +60,10 @@
       var eventUrl = $(this).data("url");  
       loadInfo(eventUrl)
     })
-  
+    $(document).ready(function() {
+      $firstEvent= $('.artist-event-row')[0]
+      eventUrl = $($firstEvent).data("url");  
+      loadInfo(eventUrl+"?set_id=0")
```

Comment:
63: Instead of `$($(selector)[0])` you can use `$(selector).first()`.
66: Use spaces around operators.

There are undeclared variables here.  It works, but it's not correct.
And please be consistent with the quotes.